TVC_MVP/County.swift

```diff
@@ -26,6 +30,9 @@ struct CountyExtras {
     var population: Int?
     var newBorns: Int?
 
-    init() { }
+    init(population: Int = 0, newBorns: Int = 0) {
```

Comment:
Why? `population` and `newBorns` are optional, you are forcing them to not be optional by assigning `0` as default value.